pkg/scripts/tempesta_installer.sh

```diff
@@ -90,13 +87,21 @@ tfw_install_packages()
 	shift
 	files=("${@}")
 
-	echo "Downloading latest packages from github.com/$GITHUB_USER/$repo ..."
-	mkdir -p $DOWNLOAD_DIR/$repo
-
-	for file in "${files[@]}"
-	do
-		tfw_download $repo $file
-	done
+  case $DISTRO in
+	"debian-8")
+    echo "Downloading latest packages from github.com/$GITHUB_USER/$repo ..."
+    mkdir -p $DOWNLOAD_DIR/$repo
+    ;;
+	"ubuntu-20")
+    repo=""
+    echo "Downloading latest packages from github.com/$GITHUB_USER/$repo ..."
+    # echo "$DOWNLOAD_DIR/$repo"
+    mkdir -p $DOWNLOAD_DIR/$repo
+    ;;
+  *)
+		;;
```

Comment:
I think we can replace outdated Debian 8 and 9 here and in `tfw_try_distro()` with Debian 11.